images/hub/z2jh/__init__.py

```diff
@@ -0,0 +1,31 @@
+"""
```

Comment:
This can be `z2jh.py`. No need for a directory when a module is one file. In general, nontrivial code in `__init__` is good to avoid.

Reply:
Done